openlibrary/templates/lib/nav_head.html

```diff
@@ -1,27 +1,28 @@
 $def with (page)
 
 <header id="header-bar" class="header-bar">
-  <div class="hamburger-component header-dropdown">
-
-    <details>
-      <summary>
-        <img src="/static/images/menu.png" alt="additional options menu"/>
-      </summary>
-      <div class="hamburger-dropdown-component navigation-dropdown-component">
-        <ul class="dropdown-menu hamburger-dropdown-menu">
-          $if not ctx.user:
-            <li><a href="/account/login">$_("Log in")</a></li>
-            <li><a href="/account/create">$_("Sign up")</a></li>
-
-          <li><a href="/books/add">$_("Add a Book")</a></li>
-          <li><a href="/sponsorship">$_("Sponsor a Book")</a></li>
-          <li><a href="/recentchanges">$_("Recent Community Edits")</a></li>
-          <li><a href="/developers">$_("Developer Center")</a></li>
-          <li><a href="/help">$_("Help & Support")</a></li>
-        </ul>
-      </div>
-    </details>
-  </div>
+  $if not ctx.user:
+    $ loginLinks = []
+  $else:
+    $ loginLinks = [
+    $     { "href": "/account/login", "text": _("Log in") },
+    $     { "href": "/account/create", "text": _("Sign up") }
+    $ ]
+  $ moreLinks = [
+  $     { "href": "/books/add", "text": _("Add a Book") },
+  $     { "href": "/sponsorship", "text": _("Sponsor a Book") },
+  $     { "href": "/collections", "text": _("Collections") },
+  $     { "href": "/recentchanges", "text": _("Recent Community Edits") },
+  $     { "href": "/developers", "text": _("Developer Center") },
+  $     { "href": "/help", "text": _("Help & Support") }
+  $ ]
+  $ hamburgerProps = {
+  $  'name': 'hamburger',
+  $   'links': loginLinks + moreLinks, # todo add links
```

Comment:
```suggestion
  $  'links': loginLinks + moreLinks, # todo add links
```
Is this `todo` still valid?

Reply:
Nope! Cleaned up whitespace and todo